ui/app/src/views/home/ProjectsAndDashboards.tsx

```diff
@@ -89,6 +89,9 @@ function RenderDashboardList(props: RenderDashboardListProps) {
               dashboardList={row.dashboards}
               hideToolbar={true}
               initialState={{
+                pagination: {
+                  paginationModel: { pageSize: 25, page: 0 },
```

Comment:
Added this as a workaround for e2e failures caused by temp dashboards not showing on the first page. Increasing pageSize from 10 to 25 ensures those dashboards can be easily accessed without having to click the next arrow (more info [here](https://github.com/perses/perses/pull/1165#issuecomment-1550557092))

Reply:
oh can you also search for the dashboard?